test/e2e/tests/bridge/bridge-cross-chain-swap.spec.ts

```diff
@@ -50,6 +50,7 @@ describe('Bridge tests', function (this: Suite) {
   ) {
     // Navigate to Bridge page
     const homePage = new HomePage(driver);
+    await homePage.check_expectedBalanceIsDisplayed();
```

Comment:
if we wait for the balance to be displayed, this will implicitly ensure that we are in the correct network and it's then picked up by the bridge condition to redirect us to the "inside Bridge" page, instead of the portfolio